### ISSUE TITLE
Support for Translation Strings

### ISSUE THREAD

**Issue by kskrlin** [micheleangioni/laravel-js-lang-converter#14]
Does this support Translation Strings (the ones which are by default saved in resources/lang as xy.json)?

I tried this package with default config and I didn't find my Translation Strings in messages.js file.

So, something like this won't work Lang.get('Register'), with de.json file:
```
{
    "Register": "Registrieren",
    "Password": "Passwort",
    "Login": "Einloggen"
}
```

**Comment by micheleangioni** [comment]
Nope, at the moment .json files are not supported. Somewhere in the future support will be added, feel free to contribute :)
